### PR TITLE
Updated sample distribution panel for figure 1

### PR DESCRIPTION
**Pull request by sjspielman**
#### What was your approach?

We need a new approach for visualizing the sample counts and their respective broad histologies, cancer groups, and tumor descriptors. Previously a new plot was made in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1201, but the labeling will prohibit the figure from fitting into a panel in main text Figure 1. 

This new approach "cleans up" the previous version for improved legibility in the main text. Notably, this is partly achieved by using _abbreviations_ for the cancer groups where appropriate, and the abbreviations have also been integrated into `mapping-histology-labels.Rmd`, which in turn added a new column `cancer_group_abbreviations` to `broad_histology_cancer_group_palette.tsv`. Those abbreviations are now also shown in the cancer group legend stub within `mapping-histology-labels.Rmd`.

#### What GitHub issue does your pull request address?

#1175 

#### Which areas should receive a particularly close look?

+ Are we ok fully integrating abbreviations into the notebook and palette TSV? My sense was this is preferred so that we can track the abbreviations _as part of the legend_ for cancer groups. The hope is this change DOES NOT break the build, and we will find out soon here.
+ Any figure aesthetics we want to discuss

#### Is there anything that you want to discuss further?

N/A

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?

Yes

#### What types of results are included (e.g., table, figure)?

+ Updated palette notebook and histology palette file to contain abbreviations
+ Figure 1 sample distribution panel

#### What is your summary of the results?

N/A

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

### CODE REVIEW

File: figures/scripts/fig1-sample-distribution.R [hunk 1]
```diff
@@ -171,36 +139,69 @@ broad_histologies <- palette_df %>%
...
+abbr_check <- data_descriptor_plot %>%
+  filter(is.na(cancer_group_abbreviation)) %>%
+  pull(cancer_group_display)
+if (!(all(abbr_check == "Other"))) stop("Wrangling bug setting cancer group abbreviations.")
+
+#make the plot
+descriptor_plot <- data_descriptor_plot %>%
+  mutate(
+    cancer_group_abbreviation = if_else(is.na(cancer_group_abbreviation), "Other", cancer_group_abbreviation),
+    bh_strip = stringr::str_wrap(broad_histology_display, 25),
```
Comment (jaclyn-taroni) [inline figures/scripts/fig1-sample-distribution.R:176]: TIL about `str_wrap()`!

File: figures/scripts/fig1-sample-distribution.R [hunk 2]
```diff
@@ -171,36 +139,69 @@ broad_histologies <- palette_df %>%
...
+             nrow = 2) +
+  labs(
+    x = "Cancer group",
+    y = "Number of samples", 
+    fill = "Tumor descriptor"
+  ) +
+  ggpubr::theme_pubr() +
+  theme(axis.text.x = element_text(size = 7.5),
+        axis.text.y = element_text(size = 10),
+        strip.text = element_text(size = 9))
```
Comment (jaclyn-taroni) [inline figures/scripts/fig1-sample-distribution.R:196]: I might rotate the x-axis labels 45 degrees. I think that might help with instances like `Schwannoma`.

**Review by jaclyn-taroni — APPROVED** [review]
I had a few minor comments. Approving with the assumption that we'll be able to come to an agreement re: the x-axis labels

**Comment by sjspielman** [comment]
Wait a second.. I accepted the commit with the changed `=` to `<-`, but was that right? I think the `=` _was_ correct. Line 185 is not creating a new variable, but a column in a df.

**Comment by jaclyn-taroni** [comment]
Oops, diffs are hard!

**Comment by sjspielman** [comment]
Here's a figure with angled labels. It looks funky for the facets with only 1 subgroup, but cleaner for the multiple subgroups eg fitting `Schwannoma` label. I'm pretty neutral on angled or not angled for this one.
[tumor_descriptor_proportion_panel.pdf](https://github.com/AlexsLemonade/OpenPBTA-analysis/files/8118441/tumor_descriptor_proportion_panel.pdf)

**Comment by jaclyn-taroni** [comment]
My slight preference is angled.

**Comment by sjspielman** [comment]
Closes #1175.

**Pull request merged**